lesson_27/pablo_lesson27/src/pages/Home/components/ProgramList/NewPrograms.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useState} from 'react';
+import {Link} from 'react-router-dom';
+
+const NewPrograms = () => {
+  const [id, setId] = useState('');
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+
+  const ProgramSubmission = (e: React.FormEvent) => {
```

Comment:
After your fetch, it would make a lot of sense to redirect back to the homepage or to some other confirmation page.